folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -332,13 +365,15 @@ private void updateResolver() {
    * <p>The optimal value should be matched to your workload and roughly the same value as your
    * memcached server's `-R` argument, which defaults to 20.
    *
-   * @param batchSize the maximum number of operations per batched client request. Default is
-   *     {@value Settings#DEFAULT_BATCH_SIZE}.
+   * @param eventLoopThreadFlushMaxBatchSize the maximum number of requests that will be written to
+   *     a connection in a single operation when sending requests on the {@link EventLoopGroup}
+   *     thread that is handling the connection. Default is {@value Settings#DEFAULT_BATCH_SIZE}.
    * @return itself
    */
-  public MemcacheClientBuilder<V> withRequestBatchSize(final int batchSize) {
-    checkArgument(batchSize > 0, "batch size must be > 0");
-    this.batchSize = batchSize;
+  public MemcacheClientBuilder<V> withEventLoopThreadFlushMaxBatchSize(
```

Comment:
I don't think we can rename the method without breaking the API

Reply:
`withRequestBatchSize` is still there. It just looks like it was renamed in the diff.